Detections/Tenable/Tenable - CriticalHigh Vulnerability found.yaml

```diff
@@ -0,0 +1,20 @@
+id: 6c0e1614-d41c-4c8f-b5f1-8b13e52c61da
+name: Tenable - Critical/High Vulnerability found
+description: |
+  'This query identifies a Tenable Vulnerability of Critical/High Vulnerability.'
+severity: High
+requiredDataConnectors:
+  - connectorId: Tenable
+    dataTypes: 
+      - Tenable_Assets_List
+queryFrequency: 12h
+queryPeriod: 12h
+triggerOperator: gt
+triggerThreshold: 0
+query: |
+
+  Tenable_Assets_List
+  | join (Tenable_Vulns_List) 
+      on AssetID
+  | project Timestamp=TimeGenerated1, AssetID, FQDN, IP, Operating_system, VulnID, VulnName, plugin_family, SeverityType, Detection_source
+  | where SeverityType in ("Critical","High")
```

Comment:
Generally you will want the where clause above the project for performance reasons.  I don't see any reason to NOT move it above the project here.